### DISCUSSION TITLE
Laravel Breeze

### DISCUSSION THREAD

**Discussion by LukaLatkovic** [Laravel-Lang/common#54] [Q&A] [answered]
There is a new package Taylor shipped a day ago [Laravel Breeze](https://github.com/laravel/breeze). It is a minimal starting point for authentication. It is lite version of Jetstream.
I believe that it uses already auth.php for language, but maybe it changes if taylor decide to upgrade package more in the future.
This is just an information :)

**Comment by suyar** [comment]
The `auth.php` language file has been updated in `laravel v8.4.1`.
see [laravel commit](https://github.com/laravel/laravel/commit/b54ef297b3c723c8438596c6e6afef93a7458b98).
So, translation files should be updated synchronously.

**Comment by luisprmat** [comment, marked as answer]
Laravel Breeze uses the existing translations, for now, so we shouldn't change anything

**Reply by luisprmat** [reply]
Except `auth.password` in `auth.php`

**Comment by luisprmat** [comment]
I made a new [pull request](https://github.com/Laravel-Lang/lang/pull/1442) to add this new translation in `auth.php`

**Comment by andrey-helldar** [comment]
My PR: https://github.com/Laravel-Lang/lang/pull/1445

**Comment by luisprmat** [comment]
New translation tag used by [Laravel Breeze](https://github.com/laravel/breeze):
* PR: [This is a secure area of the application. Please confirm your password before continuing.](https://github.com/Laravel-Lang/lang/pull/1452)
